mobile/Screens/SignUpScreen.jsx

```diff
@@ -0,0 +1,73 @@
+import { React, useState } from 'react';
+import {
+  View, TextInput, Button, Alert,
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+export default function SignUp({ navigation }) {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmedPassword, setConfirmedPassword] = useState('');
+
+  const validateEmail = (text) => {
+    const reg = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w\w+)+$/;
+    if (reg.test(text) === false) {
+      return false;
+    }
+    return true;
```

Comment:
**We can replace:**
```
if (reg.test(text) === false) {
      return false;
    }
return true;
```
    
**With:** 
`return (reg.test(text));`

While both return statements work, the latter is less redundant & helps with readability!